packages/bruno-app/src/utils/codegenerator/auth.js

```diff
@@ -0,0 +1,29 @@
+import get from 'lodash/get';
+
+export const getAuthHeaders = (collectionRootAuth, requestAuth) => {
+  const auth = collectionRootAuth && ['inherit', 'none'].includes(requestAuth.mode) ? collectionRootAuth : requestAuth;
+
+  if (auth.mode === 'basic') {
```

Comment:
Any plans on supporting other auth methods

Reply:
it's based on my expirence with insomnia, only basic and bearer works. But in the future we can add other methods